test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# First, build the binary
+source ./build.sh
+
+# Then run all test files
+for TEST in $(dirname $(find -name "*.go" | grep -v vendor | grep -v integration | grep _test.go) | uniq | cut -c3-); do
+	echo $TEST
+	go test -v ${REPO_PATH}/${TEST}
+done
+
+# And run integration tests
+go test ./integration --tags=integration
```

Comment:
These won't pass in travis yet, they need access to virtualbox right now. Probably best to leave this out for now.